src/Atoms/InfoLabel/InfoLabel.js

```diff
@@ -38,14 +38,8 @@ const InfoLabel = ({ classInfo, dogInfo, pointOnTimeLine, colorMotive }) => {
 };
 
 InfoLabel.propTypes = {
-  classInfo: PropTypes.shape({
-    dogsAmount: PropTypes.number,
-    isCompleted: PropTypes.bool,
-  }),
-  dogInfo: PropTypes.shape({
-    exercisesCompleted: PropTypes.number,
-    exercisesAmount: PropTypes.number,
-  }),
+  classInfo: PropTypes.object,
+  dogInfo: PropTypes.object,
```

Comment:
use this propTypes form development
![image](https://user-images.githubusercontent.com/75560322/152644049-605d51d4-6420-44b1-840d-31622caa0fcd.png)
(look at development)